bin/etc-update

```diff
@@ -631,6 +631,10 @@ do_merge() {
 					else
 						chown --reference="${ofile}" "${mfile}"
 						chmod --reference="${ofile}" "${mfile}"
+						${selinux} && {
```

Comment:
Please use an `if` block instead. Or get rid of the special warning and consolidate this on a single line.